src/client.js

```diff
@@ -597,29 +598,31 @@ class Client extends EventEmitter {
     // Private functions
     //
     _send(message) {
+        const preparedMessage = { ...message };
```

Comment:
Do we really need this style? Please clarify =)

Reply:
Sure, there are two ways to remove eslint errors here:
1. create a temporary variable which we will modify without eslint errors.
2. disable the eslint rule `no-param-reassign`, which prevents us from changing function arguments (that is fine too for my opinion + you can remove this temporary variable, without it the code is really cleaner).

Reply:
See https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments if we really need this. But let's try to solve these things for now. Don't skip them. =)